src/pages/projects.js

```diff
@@ -357,6 +360,10 @@ const ProjectDescriptions = () => (
             <a href="https://github.com/iovisor/gobpf">gobpf</a> is a CGo-based
             library which provides Go bindings for the BCC framework as well as
             low-level routines to load and use eBPF programs from ELF files.
+            <a href="https://github.com/aquasecurity/libbpfgo">libbpfgo</a> is a
+            Go wrapper around libbpf. It supports CO:RE and it's goal is to be a 
```

Comment:
it's BPF CO-RE, not "CO:RE", please update

s/it's/tis/

Reply:
tis or its?

Reply:
its, of course :)